src/components/LoadMore.jsx

```diff
@@ -22,6 +25,8 @@ const LoadMore = props => (
 LoadMore.propTypes = {
   onClick: PropTypes.func.isRequired,
   disabled: PropTypes.bool.isRequired,
+  isSmall: PropTypes.bool.isRequired,
```

Comment:
Remove these properties, instead define on optional className property of type String